src/containers/About/About.js

```diff
@@ -0,0 +1,161 @@
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { Table } from '@tektoncd/dashboard-components';
+import { InlineNotification } from 'carbon-components-react';
+import { getInstallProperties } from '../../api';
+import { isWebSocketConnected } from '../../reducers';
+
+const initialState = {
+  dashboardInfo: {},
+  isLoaded: false,
+  isNotFinished: true,
+  error: ''
+};
+
+export /* istanbul ignore next */ class About extends Component {
+  constructor(props) {
+    super(props);
+    this.state = initialState;
+  }
+
+  componentDidMount() {
+    this.fetchDashboardInfo();
+  }
+
+  setErrorState(errorsFound) {
+    const errrosFoundList = `${errorsFound} cannot be found`;
+
+    if (this.state.isNotFinished) {
+      this.setState({
+        error: errrosFoundList,
+        isNotFinished: false
+      });
+    }
+  }
+
+  async fetchDashboardInfo() {
+    const dash = await getInstallProperties();
+    this.setState({
+      dashboardInfo: dash,
+      isLoaded: true
+    });
+  }
+
+  makeLines(property) {
+    let data = '';
+    const value = this.state.dashboardInfo[property];
+    if (this.state.dashboardInfo[property] !== undefined) {
+      data = {
+        id: property,
+        property,
+        value
+      };
+    }
+    return data;
+  }
+
+  render() {
+    const { intl, loading } = this.props;
+    const initialHeaders = [
+      {
+        key: 'property',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.property',
+          defaultMessage: 'Property'
+        })
+      },
+      {
+        key: 'value',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.value',
+          defaultMessage: 'Value'
+        })
+      }
+    ];
+    const initialRows = [];
+    if (this.state.dashboardInfo !== '' && this.state.isLoaded === true) {
+      const errorsToDisplay = [];
+      const propertiesToCheck = [
+        'InstallNamespace',
+        'DashboardVersion',
+        'PipelineVersion'
+      ];
+      propertiesToCheck.forEach(element => {
+        const line = this.makeLines(element);
+        if (line.value !== undefined && line.value !== '') {
+          initialRows.push(line);
+        } else {
+          errorsToDisplay.push(element);
+        }
+      });
+
+      if (errorsToDisplay.length !== 0) {
+        this.setErrorState(errorsToDisplay);
+      }
+
+      return (
+        <>
+          {this.state.error !== '' && (
+            <InlineNotification
+              kind="error"
+              title={intl.formatMessage({
+                id: 'dashboard.displayVersion.error',
+                defaultMessage: 'Error getting version'
+              })}
+              subtitle={intl.formatMessage({
+                id: 'dashboard.displayVersion.errorVersions',
+                defaultMessage: this.state.error
```

Comment:
Can't use variables here, it must be a string literal otherwise there's no way to know what value should be extracted to the messages bundle for translation.

If we're generating `this.state.error` ourselves we should use `react-intl` at the point the message is created, and just pass it as a prop here. If it's an API response, we can just display it as-is.

Reply:
`this.state.error` is dynamically set based upon which ones failed in the setErrorState(errorsFound) function 
The errors message appears like `DashboardVersion,PipelineVersion cannot be found` 
DashboardVersion and PipelineVersion are the properties of the api result to getProperties function that you suggseted further up
{
  installNamespace: 'tekton-pipelines',
  dashbaordVersion: '0.5.0',
    pipelineVersion: '0.10.1'
  }
}

I have tried putting in into intl-react when its created like
```
const errorsFoundList = intl.formatMessage({
      id: 'dashboard.error.error',
      defaultMessage: `${errorsFound} cannot be found`
    });
```

Also tried doing react-intl in 
```
this.setState({
        error: intl.formatMessage({
          id: 'dashboard.error.error',
          defaultMessage: errrosFoundList
        }),
```
When i run the `npm run i18n:extract` it fails with `React Intl] Messages must be statically evaluate-able for extraction.`

Reply:
yes that 'Messages must be statically evaluate-able for extraction.' means you can't use a variable as the value of the defaultMessage, however you can use placeholders within the string as we do in other places in the UI.

Taking your first example and updating it to use a placeholder:
```diff
const errorsFoundList = intl.formatMessage({
  id: 'dashboard.error.error',
-  defaultMessage: `${errorsFound} cannot be found`
+  defaultMessage: '{errorsFound} cannot be found'
+}, {
+  errorsFound
});
```